go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/terraform-providers/terraform-provider-nsxt
 require (
 	github.com/google/uuid v1.1.1
 	github.com/hashicorp/go-version v1.2.0
-	github.com/hashicorp/terraform-plugin-sdk v1.1.0
+	github.com/hashicorp/terraform-plugin-sdk v1.12.0
 	github.com/vmware/go-vmware-nsxt v0.0.0-20191219213550-f4221331f638
 	github.com/vmware/vsphere-automation-sdk-go/lib v0.2.0
 	github.com/vmware/vsphere-automation-sdk-go/runtime v0.2.0
 	github.com/vmware/vsphere-automation-sdk-go/services/nsxt v0.3.0
+	github.com/vmware/vsphere-automation-sdk-go/services/nsxt-gm v0.1.0 // indirect
```

Comment:
Dumb question: Why is nsxt-gm indirect here?

Reply:
I'm guessing because we're not using it yet. We'll need to remove that.